Invision-README.md

```diff
@@ -0,0 +1,34 @@
+
+# Overview
+We're maintaining a fork of this repo in order to support certain
+deviations from the saml spec that are not of general use. The goal
+is to support our requirements with a minimum of maintenance and stay
+as close to upstream as possible.
+
+
+## Branches
+- `main` - This matches upstream's main exactly
+- `invision-main` - This branch includes upstream's main as well as
+    InVision-specific changes rebased on top of upstream.
+- `releases/<date>` -
+
+
+### How to make a change to this saml library
+1. Is it a bug in upstream? Submit there and update this fork when it is released.
+2. If it's InVision specific, branch from and open a PR against `invision-main`.
+    **Be careful opening the PR. Github defaults to the base fork and will open it on upstream unless you specify.**
+3. Once the PR is approved and merged, follow the process below to release.
```

Comment:
I believe this will still subject all of our own changes to a PR and Code Review, while the "update from upstream" is a separate process that only brings the latest crewjam code into our repo. This feels like a nice way to have our normal process for our code, including reviews and an audit trail while minimizing the burden on the bringing of external improvements into our code over time.